src/intents/Declaration.ts

```diff
@@ -101,6 +102,18 @@ class Declaration {
     return bits
   }
 
+  nonces (): BitNoncePair[] {
```

Comment:
we already have a `bits` method. i wonder if we should try to clean this a bit. maybe reuse some part of `bits`? or just keep the `nonces` method?

Reply:
The idea here is to return all data directly and prevent to do 
Main idea here is to have splitter logic, but then , the user need to do :
```javascript
const bits = declaration.bits()
const nonces = declaration.nonces()
```

Maybe add like this is better? 
```javascript
const bitAndNonces = declaration.bitAndNonces() // Mixed bit and nonces
const nonces = declaration.nonces() // only nonces
```

Reply:
> The idea here is to return all data directly and prevent to do
> Main idea here is to have splitter logic, but then , the user need to do :

sorry i didn't understand the explanation 😅 

> const bitAndNonces = declaration.bitAndNonces() // Mixed bit and nonces

yeah or `BitNoncePairs` to match with the type name.